cyclonedx-bom/src/models/signature.rs

```diff
@@ -0,0 +1,115 @@
+/*
+ * This file is part of CycloneDX Rust Cargo.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+use std::str::FromStr;
+
+/// Enveloped signature in [JSON Signature Format (JSF)](https://cyberphone.github.io/doc/security/jsf.html)
+#[derive(Debug, PartialEq, Eq)]
+pub struct Signature {
+    /// Signature algorithm.
+    pub algorithm: Algorithm,
+    /// The signature data.
+    pub value: String,
+}
+
+/*
+/// Enveloped signature in [JSON Signature Format (JSF)](https://cyberphone.github.io/doc/security/jsf.html)
+#[derive(Debug, PartialEq, Eq)]
+pub enum Signature {
+    /// Multiple signatures
+    Signers(Vec<Signer>),
+    /// A single signature chain
+    Chain(Signer),
+    /// A single signature
+    Signature(Signer),
+}
+
+/// For now the [`Signer`] struct only holds algorithm and value
+#[derive(Debug, PartialEq, Eq)]
+pub struct Signer {
+    /// Signature algorithm.
+    pub algorithm: Algorithm,
+    /// The signature data.
+    pub value: String,
+}
+*/
```

Comment:
What's up with the commented-out code? Ideally it should be either removed, or have a really good explanation why it's disabled in a comment next to it.